### ISSUE TITLE
Sending Failed- AWS API SES( Using Queue and not "Send Immediately" ) in Mautic 3.1

### ISSUE THREAD

**Issue by dilshadh** [mautic/mautic#9156]
Mautic Version : 3.1
Php : 7.2
Description:  I tried to send an email using AWS SES API with Queue file system option and got below error:

In AmazonApiTransport.php line 195:                                                                              
  Attempted to call an undefined method named "getAttachments" of class "Swift_Message"

### Steps to reproduce:
1. In configuration change email settings-> service configuration to AWS API. 
2. Change how email should be handled to Queue.
3. Adding  credentials to connect to ses
4. Test connection shows success
5. Send test email-pop  shows as email send
6. Logging into /var/www/html/mautic/bin. 
7. sudo php console mautic:emails:send
    After step 7. it shows the error:
                In AmazonApiTransport.php line 195:                                                                              
  Attempted to call an undefined method named "getAttachments" of class "Swift_Message"

Log errors:  
-    [2020-08-30 17:41:23] mautic.NOTICE: Symfony\Component\Debug\Exception\FatalThrowableError: Call to undefined method Swift_Message::getAttachments() (uncaught exception) at /var/www/html/mautic/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonApiTransport.php line 195 while running console command `mautic:emails:send` [] []
-   [2020-08-30 17:41:23] mautic.WARNING: Command `mautic:emails:send` exited with status code 1 [] []

**Comment by mabumusa1** [comment]
@dennisameling Maybe you can help us with this?

When this [function](https://github.com/mautic/mautic/blob/2cdc757e8744e4b11d8bb85b574c4ce9777706ff/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonApiTransport.php#L537) is called on the queue mode, the object I get is not of the type `MauticMessage` but when the emails are sent immediately it works, do you have an idea what could happen there?

**Comment by adamfasullo** [comment]
i'm also having the same issue. not sure what to do from this point.

**Comment by bshowket** [comment]
exact Same issue for me , error

`In AmazonApiTransport.php line 195:

Attempted to call an undefined method named "getAttachments" of class "Swif
t_Message".


mautic:emails:send [--message-limit [MESSAGE-LIMIT]] [--time-limit [TIME-LIMIT]] [--do-not-clear] [--recover-timeout [RECOVER-TIMEOUT]] [--c...`

**Comment by jabaco** [comment]
Same issue here: 

`2020-09-14 21:57:04] mautic.NOTICE: Symfony\Component\Debug\Exception\FatalThrowableError: Call to undefined method Swift_Message::getAttachments() (uncaught exception) at /var/www/vhosts/mautic.webpros.marketing/httpdocs/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonApiTransport.php line 195 while running console command `mautic:emails:send` [] []
[2020-09-14 21:57:04] mautic.WARNING: Command `mautic:emails:send` exited with status code 1 [] []`

**Comment by mautibot** [comment]
This issue has been mentioned on **Mautic Community Forums**. There might be relevant details there:

https://forum.mautic.org/t/amazon-ses-api-is-not-working/16078/3

**Comment by alexdhika** [comment]
same problem

**Comment by tenil** [comment]
Same problem, same Steps to reproduce. :/

**Comment by RCheesley** [comment]
Hi folks, unfortunately we were not able to get this resolved in the 3.1.1 release so I've bumped it back to 3.1.2.

If anybody would be up for looking into it please shout and we can help you get set up to make a PR!

**Comment by jabaco** [comment]
Why?? this is a crucial feature... Mautic 3.1 Main feature is Amazon SES Api integration and this should be a priority.... one month more to wait is too much...

**Comment by tenil** [comment]
> This issue has been mentioned on **Mautic Community Forums**. There might be relevant details there:
> 
> https://forum.mautic.org/t/amazon-ses-api-is-not-working/16078/3

I followed those steps and it didn't work out.

**Comment by J-Light** [comment]
Upgraded to 3.1 from 2.16 hoping to use the feature. We are having the same issues.

**Comment by bobotdls** [comment]
anybody already some solution on this issue?

**Comment by bobotdls** [comment]
> Hi folks, unfortunately we were not able to get this resolved in the 3.1.1 release so I've bumped it back to 3.1.2.
> 
> If anybody would be up for looking into it please shout and we can help you get set up to make a PR!

Yeah I need your help

**Comment by streamworksaudio** [comment]
Having the same issue here.  Was going to send our first bulk newsletter today through Mautic/SWA SES.  When we tried to send a preview test mail we got this error.  Setting it back to 'Send Immediately' allowed the mail to go though.

I would say this is an urgent bug as anybody sending email to a large list through AWS SES need to send in a queue and not all at one time or else they risk loosing their SES account.
